choc-pizza-react/src/Head.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+export default class Head extends Component {
```

Comment:
This component isn't actually necessary - the index.html has all this info already.